packages/share_plus/share_plus/example/pubspec.yaml

```diff
@@ -4,7 +4,8 @@ description: Demonstrates how to use the share_plus plugin.
 dependencies:
   flutter:
     sdk: flutter
-  share_plus: ^9.0.0
+  share_plus:
+    path: ../
```

Comment:
This should be reverted. Please take a look at the contribution guide in the repo as it has instructions on how to bootstrap the repository so the examples use the local packages (melos bootstrap)